src/main/java/org/kcctl/service/KafkaConnectApi.java

```diff
@@ -71,6 +71,13 @@ public interface KafkaConnectApi {
     @Path("/connectors/{name}/restart")
     void restartConnector(@PathParam("name") String name);
 
+    @POST
+    @Path("/connectors/{name}/restart")
+    ConnectorStatusInfo restartConnectorAndTasks(
+                                                 @PathParam("name") String name,
+                                                 @QueryParam("includeTasks") boolean includeTasks,
+                                                 @QueryParam("onlyFailed") boolean onlyFailed);
```

Comment:
I really wanted to omit the `includeTasks` parameter here and hardcode it to be `true` in the REST request, but couldn't find a way to do that. Based on https://github.com/eclipse/microprofile-rest-client/issues/315 it may not even be possible.